src/REPLMode.jl

```diff
@@ -208,6 +209,14 @@ function lex(cmd::String)::Vector{QString}
                 push_token!(false)
                 push!(qstrings, QString(";", false))
             end
+        elseif c == ','
+            if in_doublequote || in_singlequote || !replace_comma# raw char
```

Comment:
```suggestion
            if in_doublequote || in_singlequote || !replace_comma # raw char
```
I don't really follow what this comment means...

Reply:
I'm not sure but it appears several time in the lexer
https://github.com/JuliaLang/Pkg.jl/blob/6d8f0727c8809e4b06da29acda8b55e1cfa4dcb1/src/REPLMode.jl#L199

Reply:
Ok, I guess carry on then. @00vareladavid, can you explain what the comment means?

Reply:
Yeah it means "not a delimiter" i.e. a character that will eventually be used as part of some identifier. Perhaps that should be more clear